cloud/docker_client.go

```diff
@@ -81,6 +85,17 @@ func (c *dockerClientImpl) generateClient(h *host.Host) (*docker.Client, error)
 	return c.client, nil
 }
 
+func (c *dockerClientImpl) changeTimeout(h *host.Host, newTimeout time.Duration) (*docker.Client, error) {
+	c.client = nil
```

Comment:
Did you forget to reset this to the new client at the end of the function?

Reply:
Fixed.